src/styles/_components/_alerts.scss

```diff
@@ -1,23 +1,35 @@
 @mixin alert() {
+    //note: all the icons svgs are available in the images!
+    @include column(12);
+    @include justify-content-center();
     @include flex();
     @include align-items-center();
     padding: 15px 60px;
-    background-color: color('feedback', 'info');
+    background-color: color('feedback');
```

Comment:
If no second parameter is passed, it assumes the first value, in this case, 'primary'